### ISSUE TITLE
Android 11: notification can't be restored after being swiped away

### ISSUE THREAD

**Issue by nbmrjuhneibkr** [TeamNewPipe/NewPipe#4524]
### Checklist

- [x] I am using the latest version. 
- [x] I checked, but didn't find any duplicates of this issue in the repo. 
- [x] I have read the contribution guidelines given at https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md.
- [x] This issue contains only one bug. I will open one issue for every bug report I want to file.

### Steps to reproduce the bug
0. Android 11
1. Add videos to the background queue.
2. Open the notification shade.
3. Swipe away NewPipe notification (playback continues).
4a. Fully expand quick settings area to show NewPipe player controls.
4b. Fully expand quick settings area and open the background queue in NewPipe.
5. Interact with the player - play/pause, change tracks.

### Actual behaviour
NewPipe notification doesn't reappear. It will appear again only after you clear the queue and start playback again.

### Expected behavior
NewPipe notification should reappear as soon as you interact with the player controls (either from the quick settings area or within the app).  
This behavior can be seen with the Google Play Music app.

### Device info

 - Android version/Custom ROM version: Android 11
 - Device model: Pixel 3a

**Comment by opusforlife2** [comment]
@Stypox This needs both a working Android 11 emulator and the 'Save queue to disk' feature as a prerequisite, right?

**Comment by nbmrjuhneibkr** [comment]
>Save queue to disk

Why would that be needed? The queue doesn't get erased when notification is swiped away, and playback continues just fine. I still can access the queue, just not from the notification.

**Comment by opusforlife2** [comment]
Oh. Okay.

**Comment by Stypox** [comment]
When would you expect the notification to appear again? I mean, if you swiped it away on purpose it should stay hidden until the player is restarted, shouldn't it? Otherwise why would you have swiped it away? I really can't understand what the purpose of this new Android 11 feature is, and until I can't get a working emulator I can't test anything.

**Comment by nbmrjuhneibkr** [comment]
> When would you expect the notification to appear again?

=>

>NewPipe notification should reappear as soon as you interact with the player controls (either from the quick settings area or within the app). **This behavior can be seen with the Google Play Music app.**

>if you swiped it away on purpose it should stay hidden until the player is restarted, shouldn't it?

The player is not stopped, it keeps playing. So now I have to interrupt playback before I can make the notification appear again.

>Otherwise why would you have swiped it away?

On accident, while clearing other notifications, obviously.

>I really can't understand what the purpose of this new Android 11 feature is

Neither can I, but it's already here, and I doubt that we can convince Google to change it.

**Comment by nbmrjuhneibkr** [comment]
5 months later, this issue appears to be forgotten. Does anyone else use Android 11 here?

**Comment by Stypox** [comment]
@nbmrjuhneibkr this would require a burden of work to implement, so there are things with more priority taking over

**Comment by nbmrjuhneibkr** [comment]
Oh, surely this isn't a top priority issue. I just wanted to make sure that I'm not the only one who remembers it.

**Comment by Stypox** [comment]
It made me headache a while ago, I do remember it ;-)

**Comment by XiangRongLin** [comment]
#4335
With that solved this issue should be too because the described error can't situation can't happen anymore
